pkg/types/conversion/installconfig.go

```diff
@@ -52,6 +53,11 @@ func ConvertInstallConfig(config *types.InstallConfig) error {
 			return err
 		}
 	case vsphere.Name:
+		if err := vsphereconversion.ConvertInstallConfig(config); err != nil {
+			return err
+		}
+
+		// TODO: jcallen: This should be moved...
```

Comment:
Is this done? I'm not sure what it means.

Reply:
Not sure it makes sense to move now. Will remove the comment.
I was doing alot of `TODO: jcallen:` so I can find things that I needed to fix. Will make sure they are all gone.
Thanks!